Reloj_checador/app/assets/stylesheets/home.scss

```diff
@@ -51,34 +37,12 @@ html,body{
     outline: 0 0 0 0  !important;
     box-shadow: 0 0 0 0 !important;
     
-    }
-    
-    .remember{
-    color: white;
-    }
-    
-    .remember input
-    {
-    width: 20px;
-    height: 20px;
-    margin-left: 15px;
-    margin-right: 5px;
-    }
-    
+    }  
     .login_btn{
-    color: black;
     width: 100px;
-    background-color: white !important;
     }
 
     .login_btn:hover{
     background-color: rgb(226, 226, 226) !important;
     }
-    
-    .links{
-    color: white;
-    }
-    
-    .links a{
-    margin-left: 4px;
-    }
\ No newline at end of file
+    
```

Comment:
Remember to add a blank line at the end of the file, when you submit the PR github notify you about that with this sign.